contracts/apps/disputable/IAgreement.sol

```diff
@@ -0,0 +1,103 @@
+/*
+ * SPDX-License-Identifier:    MIT
+ */
+
+pragma solidity ^0.4.24;
+
+import "../../acl/IACLOracle.sol";
+import "../../lib/token/ERC20.sol";
+import "../../lib/arbitration/IArbitrable.sol";
+
+
+contract IAgreement is IArbitrable, IACLOracle {
+
+    event Signed(address indexed signer, uint256 settingId);
+    event ActionSubmitted(uint256 indexed actionId);
```

Comment:
Wondering if we should also include the disputable's address as an indexed parameter here too

Reply:
Discussed offline: we agreed on adding it since indexing this address adds ~100 gas